### ISSUE TITLE
Create Landing Page 'News worth telling you about' Section

### ISSUE THREAD

**Issue by Cleop** [charterindex/application#56]
![image](https://user-images.githubusercontent.com/16775804/38096043-ce1431e2-3369-11e8-9e8f-cd6b84531f2c.png)

**As a** user
**I want to** have access to relevant yacht related news
**So that** I can be inspired on when/where to go next and to trust the authority of the CI brand as experts in yachting

Technical note: these destination cards may appear on the 'News' page itself and therefore benefit from being made as a reusable component.

## Acceptance
- [ ] Adds 'News worth telling you about' title and subtitle
- [ ] Cards appear in rows of 2 (max) but stack on mobile (need to determine tablet view)
- [ ] Cards display a photo relating to the news story - available from the API?
- [ ] Cards display 2 details (article title and tagline, both sourced from the API?)
- [ ] Underneath the cards is a 'See more news' link (which takes you to the 'news' page)

#### Questions
- Which 2 featured news articles do you want displaying on the homepage? The 2 most recent?
- Where are we getting the news article images, article title and article tagline from, the API?

**Comment by PeterTrotter** [comment]
@Cleop Answers to questions:

> Which 2 featured news articles do you want displaying on the homepage? The 2 most recent?
The landing page will be retrievable from the API and will have a selection of yachts and different types of articles to be shown so that Chris can have a bit more control over this.

>Are images that are provided by the API/CMS a set ratio? or will we need to handle different image shapes/sizes?
Images will only ever be 16/9

>Where are we getting the news article images, article title and article tagline from, the API/CMS? If progress on this is a blocker, shall we start by hard coding fake content?
This will come from the API. Some content is being generated but I'm currently working on getting yachts delivered so it will be worth stubbing this information out if it is required now. Bear in mind that each Card will be backed by an Article or a Yacht. That's the current plan anyway...

**Comment by chriswebley** [comment]
A1: The two most recent articles will feature.
A2: Images will be hosted in the exact same 16:9 sizes (to start with)
A3: Images will be sourced from the CMS

**Comment by iteles** [comment]
Agreed in sprint planning that for this sprint we will have hard-coded images and text.

@chriswebley As discussed, could you please drop in some real data for us to use here (photos and text)? If you could aim to vary the length of the titles and straplines so we can test for various use cases, that would be great.
